File: splearn/src/test/java/com/splearn/domain/EmailTest.java
package com.splearn.domain;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;
import static org.junit.jupiter.api.Assertions.*;

class EmailTest {
    @Test
    void equality(){
        var email1 = new Email("cham@app.com");
        var email2 = new Email("cham@app.com");
        assertThat(email1).isEqualTo(email2);
    }
    

}

File: splearn/src/test/kotlin/cham/splearn/domain/EmailTest.kt
package cham.splearn.domain

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe


class EmailTest : FunSpec({

    test("equality") {
        val email1 = Email.of("cham@splearn.app")
        val email2 = Email.of("cham@splearn.app")

        email1 shouldBe email2
    }

})